FilePersistence/src/version_control/ListMergeComponent.cpp

```diff
@@ -275,6 +275,12 @@ bool onlyConflictsOnLabel(ConflictPairs& conflictPairs, QSet<Model::NodeIdType>&
 
 		if (changeA && changeB)
 		{
+			if (changeA->onlyLabelChange() && changeB->onlyLabelChange())	continue;
+			else return false;
+			qDebug() << containerId;	//	Used for the code described below
+/*			This part of code contains the original code implemented in Balz thesis
```

Comment:
Please comment this and the next line with `//` and only put the `/*` after the text, once the code starts. That way, even if someone removes the `/*` the code compiles.